app/assets/javascripts/authoring/events.js

```diff
@@ -257,6 +258,77 @@ function createEvent(point, duration) {
     updateStatus();
 };
 
+function newEventObj(snapPoint, duration){
+    duration = duration || 60;
+    var startTimeObj = getStartTime(snapPoint[0]);
+    var newEvent = {
+        "title":"New Event", "id":createEventId(),
+        "x": snapPoint[0]-4, "min_x": snapPoint[0], "y": snapPoint[1], //NOTE: -4 on x is for 1/15/15 render of events
+        "startTime": startTimeObj["startTimeinMinutes"], "duration":duration,
+        "members":[], timer:0, task_startBtn_time:-1, task_endBtn_time:-1,
+        "dri":"", "pc":"", "notes":"", "startHr": startTimeObj["startHr"], "status":"not_started",
+        "startMin": startTimeObj["startMin"], "gdrive":[], "completed_x":null, "inputs":"", "all_inputs":"", "outputs":"", events_after : "",
+        "docQs": [["Please explain all other design or execution decisions made, along with the reason they were made",""],
+        ["Please add anything else you want other team members, the project coordinator, or the client, to know. (optional)",""]],
+        "outputQs":{},"row": Math.floor((snapPoint[1]-5)/_foundry.timeline.rowHeight)};
+
+    return newEvent;
+}
+function duplicateEventObj(eventObject){
+    var newObject = JSON.parse(JSON.stringify(eventObject));
+    newObject["id"] = createEventId();
+    newObject["y"] = newObject["y"] + RECTANGLE_HEIGHT + 20;
+    newObject["row"] = Math.floor((newObject["y"]-5)/_foundry.timeline.rowHeight);
+    return newObject;
+}
+
+//task_startBtn_time and task_endBtn_time refer to the time when the start button and end button on the task is clicked.
+function createEventObj(eventObject) {
+
+    //add new event to flashTeams database
+    flashTeamsJSON.events.push(eventObject);
+
+    return eventObject;
+};
+
+function onConfigClick(event){
+    CURRENT_EVENT_SELECTED = event.groupNum;
+    showDropDown();
+}
+
+function showDropDown(){
+    var menu = $.contextMenu({
+        selector: '.icon-cog',
+        trigger: 'left',
+        callback: function(key, options) {
+            //var m = "clicked: "  key;
+            //window.console && console.log(m) || alert(options.groupNum);
+            window[key + "Event"](CURRENT_EVENT_SELECTED);
+        },
+        items: {
+            "duplicate": {name: "Duplicate", icon: ""},
+            "edit": {name: "Edit", icon: ""},
+            "confirmDelete": {name: "Delete", icon: ""}
+        }
+    });
+}
+
+function duplicateEvent(groupNumber){
+    var task_id = getEventJSONIndex(groupNumber);
+    var eventToDuplicate = flashTeamsJSON["events"][task_id];
+
+    var eventObj = createEventObj(duplicateEventObj(eventToDuplicate));
+
+    drawEvent(eventObj, true);
+
+    // save
+    updateStatus(false);
```

Comment:
updateStatus should not have a parameter. It should just be updateStatus();

The reason for this is because events can be created (and duplicated) once the team has started (via edit mode). See my comments on your PR for additional details.